Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java

```diff
@@ -453,14 +448,10 @@ public void emailUser(CallingContext context, String userName, String emailTempl
         RaptureUser user = getUser(context, userName);
         if (user != null) {
             templateValues.put("user", user);
-            Mailer.email(context, emailTemplate, templateValues);
-        } else {
-            throw RaptureExceptionFactory.create(HttpURLConnection.HTTP_BAD_REQUEST, adminMessageCatalog.getMessage("NoExistUser", userName)); //$NON-NLS-1$
         }
+        Mailer.email(context, emailTemplate, templateValues);
```

Comment:
In my test case I didn't have a user so it barfed unnecessarily. 
This was a quick fix; I can fix my test to add a user.